app/assess/templates/get_help.html

```diff
@@ -0,0 +1,48 @@
+{% extends "base.html" %}
+
+{%- from 'govuk_frontend_jinja/components/back-link/macro.html' import govukBackLink -%}
+{% from "macros/logout_partial.html" import logout_partial %}
+
+{% block content %}
+
+    </div> {# This div is here on purpose. #}
+    {% if g.user %}
+        <div class="fsd-login-banner-background govuk-!-margin-bottom-0"></div>
+        <div class="govuk-width-container">
+            <div class="govuk-phase-banner flex-parent-element">
+                {% if request.referrer %}
+                    <p class="govuk-!-text-align-left flex-parent-element flexed-element-margins-collapse">
+                        {{ govukBackLink({'href': request.referrer, 'text': 'Back'}) }}
+                    </p>
+                {% endif %}
+
+                {{ logout_partial(sso_logout_url, show_get_help=False) }}
+            </div>
+        </div>
+    {% else %}
+        <div class="fsd-login-banner-background"></div>
+    {% endif %}
+    </div>
+    <div class="govuk-width-container">
+        <h2 class="govuk-heading-l">Get help</h2>
```

Comment:
Can we make this an `<h1>`? Given I don't _think_ there's an `<h1>` above this on the page?

Reply:
Good spot, I've changed in https://github.com/communitiesuk/funding-service-design-assessment/pull/175/commits/5e8f874d4caf83c82da3546a2f5de7a84df3a1f7